components/sanity/scripts/create.py

```diff
@@ -0,0 +1,17 @@
+#!/bin/python
+
+import fabric.api
+
+from cloudify import ctx
+
+
+def upload_keypair(manager_user, local_key_path):
+    ctx.logger.info('putting key from {0}'.format(local_key_path))
+    manager_remote_key_path = '/home/{0}/.ssh/mng-key.pem'.format(manager_user)
```

Comment:
this is not good.. /home/USERNAME is only true for specific use cases. For instance.. root's home is /root/